tools/packaging/kata-deploy/nodefeaturerules/tdx-nfd-rule.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: nfd.k8s-sigs.io/v1alpha1
+kind: NodeFeatureRule
+metadata:
+  name: tdx-total-keys
+spec:
+  rules:
+  - name: "TDX total keys rule"
+    labels:
+      "intel.feature.node.kubernetes.io/tdx": "true"
+    extendedResources:
+      tdx.intel.io/keys: "@cpu.security.tdx.total_keys"
```

Comment:
@fidencio could you explain to a newbas like me what "TEE keys book keeping" is meant for and how it works?

Reply:
When we start a VM using TEE capabilities, one key is used to do the memory encryption.
The hardwares have a limitied amount of keys, and those **must** be managed, in order to ensure that containers will land in nodes that do have keys available.

Makes sense?

Reply:
Got it. thanks!